web/src/main/java/se/inera/intyg/webcert/web/web/controller/testability/facade/util/CreateCertificateTestabilityUtil.java

```diff
@@ -142,9 +148,13 @@ public String createNewCertificate(@NotNull CreateCertificateRequestDTO createCe
 
         final var certificate = utkastToCertificateConverter.convert(utkast);
 
-        updateCertificate(createCertificateRequest, certificate);
-
-        utkast.setModel(getJsonFromCertificate(certificate, utkast.getModel()));
+        if (certificate.getMetadata().getType().equalsIgnoreCase(Fk7263EntryPoint.MODULE_ID)) {
```

Comment:
Change based on our discussion.

Here is a code snippet from our discussion to act as a reminder.

        if (utkast.getIntygsTyp().equalsIgnoreCase(Fk7263EntryPoint.MODULE_ID)) {
            try {
                final var moduleApi = moduleRegistry.getModuleApi(
                    utkast.getIntygsTyp(),
                    utkast.getIntygTypeVersion()
                );

                final var utlatande = moduleApi.getUtlatandeFromJson(utkast.getModel());
                updateUtlatande(createCertificateRequest, utlatande);
//                final var utlatande = getUtlatande(createCertificateRequest, certificate);
                utkast.setModel(getJsonFromUtlatande(utlatande));
            } catch (Exception e) {
                throw new RuntimeException(e);
            }
        } else {
            final var certificate = utkastToCertificateConverter.convert(utkast);
            updateCertificate(createCertificateRequest, certificate);
            utkast.setModel(getJsonFromCertificate(certificate, utkast.getModel()));
        }